### PR TITLE
lmbench.py Showing the results

### PR DESCRIPTION
**Pull request by nasastry**
This patch shows the results with 'make see'.

Signed-off-by: Nageswara R Sastry <rnsastry@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by nasastry** [comment]
```
# avocado run lmbench.py
JOB ID     : c39112bd8641d0533cdc89ecf61c9e8278b33510
JOB LOG    : /home/nasastry/avocado-fvt-wrapper/results/job-2018-06-29T03.47-c39112b/job.log
 (1/1) lmbench.py:Lmbench.test: -Using config in CONFIG.linux-27yb
Fri Jun 29 03:47:43 EDT 2018
Latency measurements
-Fri Jun 29 03:48:15 EDT 2018
Calculating file system latency
\Fri Jun 29 03:48:17 EDT 2018
Local networking
/Fri Jun 29 03:49:59 EDT 2018
Bandwidth measurements
/Fri Jun 29 03:51:08 EDT 2018
Calculating context switch overhead
/Fri Jun 29 03:51:12 EDT 2018
Calculating effective TLB size
\Fri Jun 29 03:51:13 EDT 2018
Calculating memory load parallelism
|Fri Jun 29 03:51:42 EDT 2018
McCalpin's STREAM benchmark
/Fri Jun 29 03:51:44 EDT 2018
Calculating memory load latency
|Fri Jun 29 04:11:10 EDT 2018
PASS (1436.88 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1437.36 s
JOB HTML   : /home/nasastry/avocado-fvt-wrapper/results/job-2018-06-29T03.47-c39112b/results.html
```

File: perf/lmbench.py [hunk 1]
```diff
@@ -116,6 +116,8 @@ def test(self):
 
         os.chdir(self.sourcedir)
         build.make(self.sourcedir, extra_args='rerun')
+        build.make(self.sourcedir, extra_args='rerun')
```
Comment (PraveenPenguin) [inline perf/lmbench.py]: why two times , `rerun`

File: perf/lmbench.py [hunk 2]
```diff
@@ -116,6 +116,8 @@ def test(self):
...
+        build.make(self.sourcedir, extra_args='see')
```
Comment (PraveenPenguin) [inline perf/lmbench.py]: thinking it can be yaml param ?

File: perf/lmbench.py (hunk 1 above)
Comment (nasastry) [reply perf/lmbench.py]: @PraveenPenguin 
Wanted to see the aggregate of two runs ... `make see` shows the aggregate of the above two runs.

File: perf/lmbench.py (hunk 2 above)
Comment (nasastry) [reply perf/lmbench.py]: @PraveenPenguin Not required in yaml

File: perf/lmbench.py (hunk 1 above)
Comment (PraveenPenguin) [reply perf/lmbench.py]: ah sure

File: perf/lmbench.py (hunk 2 above)
Comment (PraveenPenguin) [reply perf/lmbench.py]: ok

**Pull request merged**